### PR TITLE
Port JITServer changes in J9VMEnv/J9VMMethodEnv

### PR DESCRIPTION
**Pull request by harryyu1994**
Add JITServer support for a few VM env APIs.
- J9::VMEnv::getInterpreterVTableOffset
- J9::VMMethodEnv::hasBackwardBranches
- J9::VMMethodEnv::bytecodeStart
- J9::VMMethodEnv::bytecodeSize

Signed-off-by: Harry Yu <harryyu1994@gmail.com>

### CODE REVIEW

**Comment by harryyu1994** [comment]
@mpirvu Hi Marius, this is the remaining changes for "runtime/compiler/env" to be ported to master. Ready for review.

File: runtime/compiler/env/J9VMMethodEnv.cpp [hunk 1]
```diff
@@ -29,11 +29,18 @@
...
+#if defined(JITSERVER_SUPPORT)
+#include "control/JITServerHelpers.hpp"
+#endif
 
 bool
 J9::VMMethodEnv::hasBackwardBranches(TR_OpaqueMethodBlock *method)
    {
+#if defined(JITSERVER_SUPPORT)
+   J9ROMMethod * romMethod = JITServerHelpers::romMethodOfRamMethod((J9Method *)method);
+#else
```
[the remark this answers is not in the document]
Comment (harryyu1994) [reply runtime/compiler/env/J9VMMethodEnv.cpp]: The test is inside `JITServerHelpers::romMethodOfRamMethod`

**Comment by harryyu1994** [comment]
Note to self: 85768adadcd2c4e9a4eaa9a72131f36f93328ae6 needs to be ported back to jitaas branch

**Comment by mpirvu** [comment]
Jenkins test sanity all jdk8,jdk11

**Comment by harryyu1994** [comment]
Jenkins test didn't trigger for some reason.

**Comment by mpirvu** [comment]
Jenkins test sanity all jdk8,jdk11

**Comment by harryyu1994** [comment]
https://github.com/eclipse/openj9/pull/7485 I have found some more runtime/compiler/env changes that needs porting.

**Comment by mpirvu** [comment]
Testing passed, hence merging.

**Pull request merged**